src/core/agent/services/ipexCommunicationService.types.ts

```diff
@@ -8,6 +8,7 @@ interface CredentialsMatchingApply {
     acdc: any;
   }[];
   attributes: Record<string, unknown>;
+  identifierId: string;
```

Comment:
I think we can just call it `identifier` in general. identifier = ID so identifierId is like ID ID haha